### ISSUE TITLE
Checking status of this Ansible role

### ISSUE THREAD

**Issue by sebastian-luna-valero** [usegalaxy-eu/ansible-htcondor#5]
Hello,

After https://github.com/usegalaxy-eu/vgcn/pull/56 could you confirm this Ansible role is no longer valid?

Is it worth investing effort in updating it to streamline pulsar deployments?

Best regards,
Sebastian

**Comment by kysrpex** [comment]
Hi, do you mean it should be invalid because of the following?

> external comes without HTCondor now, this is because the configuration of HTCondor changed and is best done with the [getHTCondor-Script](https://htcondor.readthedocs.io/en/latest/getting-htcondor/admin-quick-start.html#assigning-roles-to-machines). With that script you can set your own cluster password and host. This would require a lot of error-prone configuration otherwise.

I cannot answer immediately whether it is still valid or not. In addition, I do not know what

> require a lot of error-prone configuration otherwise.

precisely means. @mira-miracoli (not currently available) could easily answer your second question I believe. With more time I could have a look into the first.

**Comment by sebastian-luna-valero** [comment]
Yes, that's what I mean. Let's then wait for Mira's thoughts on this.

Thank you!

**Comment by sebastian-luna-valero** [comment]
@mtangaro could you please confirm whether you used this Ansible role to deploy HTCondor 10.x?

**Comment by sanjaysrikakulam** [comment]
At EU, we are **not** using HTCondor 10. x. We are still using the older version `8.8.15`.

**Comment by sebastian-luna-valero** [comment]
Thanks, All.

Then, apparently I misinterpreted the comments in https://github.com/usegalaxy-eu/vgcn/pull/56 and assumed this role wasn't properly maintained.

In that sense, I invested some time updating a separate Ansible role for HTCondor in https://github.com/grycap/ansible-role-htcondor/pull/10. Sorry for the duplication of effort. I am planning to use this updated Ansible role for HTCondor streamline the deployment of pulsar via https://im.egi.eu/ in the context of the EuroScienceGateway project.

**Comment by bgruening** [comment]
Oh this other role was updated the last time 3 years ago. I hope they merge it. Let us know if you want to use this role here and need anything else.

**Comment by sebastian-luna-valero** [comment]
Just for the record, check also https://github.com/usegalaxy-eu/vgcn/commit/ebec04289b3d08c1d7ecd8b11f8c549dfcf4e58b

**Comment by domgz** [comment]
@bgruening @sanjaysrikakulam Actually why don't we switch to [grycap/ansible-role-htcondor](https://github.com/grycap/ansible-role-htcondor) also at [usegalaxy.eu](https://usegalaxy.eu/)? I have been doing some work on our role before realizing the other role exists, but honestly this other role is still better than ours + new changes.

If we are happy with that other role I suggest we even archive this repository.

**Comment by bgruening** [comment]
@kysrpex it was hardly maintained for 3 years. If its back to live and we can use it, lets do it.

**Comment by domgz** [comment]
> @kysrpex it was hardly maintained for 3 years. If its back to live and we can use it, lets do it.

That role seems to do things that ours + new changes does not, most importantly installing two roles on the same machine (needed for the headnode) and supporting the "release" and "stable" channels. If I commit what I have it will be a half-assed solution compared to said role. I am giving it a try, if it does not work we can always continue using ours.

**Comment by domgz** [comment]
> > @kysrpex it was hardly maintained for 3 years. If its back to live and we can use it, lets do it.
> 
> That role seems to do things that ours + new changes does not, most importantly installing two roles on the same machine (needed for the headnode) and supporting the "release" and "stable" channels. If I commit what I have it will be a half-assed solution compared to said role. I am giving it a try, if it does not work we can always continue using ours.

Just FYI, after looking at the role I am not so sure after all. They're quite different approaches... I'll update you with whatever I do anyway.

**Comment by domgz** [comment]
We are using now [kysrpex/ansible-htcondor-grycap](https://github.com/kysrpex/grycap-ansible-role-htcondor) (but should be using [usegalaxy-eu/ansible-htcondor-grycap](https://github.com/usegalaxy-eu/ansible-htcondor-grycap), see usegalaxy-eu/infrastructure-playbook#1234).

Considering that, I would suggest archiving this repository.
